cmd/usd-mailer/main.go

```diff
@@ -14,10 +14,10 @@ func main() {
 	pkg.LoadConfig()
 	database.ConnectDatabase()
 
-	database.DB.AutoMigrate(&models.Email{})
-
+	if err := database.DB.AutoMigrate(&models.Email{}); err != nil {
```

Comment:
Reccomendation here to but `err != nil {` on the next row to improve readability, but not sure if its possible by Go syntax or if it is regular codestyle for platform

Reply:
It is a common way to work with errors if the only return value of the function is an error